### DISCUSSION TITLE
two 6600 and two 560

### DISCUSSION THREAD

**Discussion by aladrocMatiner** [ROCm/ROCm#3103] [Q&A] [answered]
Hej, Ive been reading around how to make this happened and I used the 6.1.0 to install but i heard 5.7.1 is the last distro that supports legacy cards like 560... so i installed both:

```
sudo amdgpu-install --usecase=workstation,rocm,graphics,opencl --opencl=rocr,legacy --vulkan=amdvlk,pro --rocmrelease=6.1 --no-32 -y --accept-eula
sudo amdgpu-install --usecase=workstation,rocm,graphics,opencl --opencl=rocr,legacy --vulkan=amdvlk,pro --rocmrelease=5.7.1 --no-32 -y --accept-eula
```
On the other side his is how my lspci -k looks like
```
03:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Navi 23 [Radeon RX 6600/6600 XT/6600M] (rev c1)
        Subsystem: XFX Limited Navi 23 [Radeon RX 6600/6600 XT/6600M]
        Kernel driver in use: amdgpu
        Kernel modules: amdgpu
06:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Navi 23 [Radeon RX 6600/6600 XT/6600M] (rev c1)
        Subsystem: XFX Limited Navi 23 [Radeon RX 6600/6600 XT/6600M]
        Kernel driver in use: amdgpu
        Kernel modules: amdgpu
07:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Baffin [Radeon RX 550 640SP / RX 560/560X] (rev c1)
        Subsystem: ASUSTeK Computer Inc. Baffin [Radeon RX 550 640SP / RX 560/560X]
        Kernel driver in use: amdgpu
        Kernel modules: amdgpu
08:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Baffin [Radeon RX 550 640SP / RX 560/560X] (rev c1)
        Subsystem: ASUSTeK Computer Inc. Baffin [Radeon RX 550 640SP / RX 560/560X]
        Kernel driver in use: amdgpu
        Kernel modules: amdgpu
```

I see all have same module but both belong to gfx1032 (6600) platform and gfx803 (560) and I wonder how i can see if the right driver has been loaded since the rocm-smi only shows the 6600 cards but not the 560 does not show up...

what im doing wrong?

**Comment by zichguan-amd** [comment, marked as answer]
Hi @aladrocMatiner, you can only install one version for the kernel driver as documented here: https://rocm.docs.amd.com/projects/install-on-linux/en/develop/install/native-install/multi-version-install.html. It's normal that all entries listed by `lspci` show the same kernel driver. You can view and configure different ROCm versions following theses instructions.
